README.md

```diff
@@ -23,9 +23,11 @@ AudienceStream allows you to create a unified view of your customers, correlatin
 
 ## Change Log
 
-- 1.1.1  Bug Fix
+- 1.2.0 Support for "datasource" and UDH
+   - Add "datasource" as optional argument to constructor
+   - Make "environment" optional. It's no longer required, but won't break to use.
```

Comment:
Minor: Keep spacing between versions the same, I believe one line between each is the current pattern.